TRANSFORMERS/TEXT_MANIPULATION/TEXT_CONCAT/TEXT_CONCAT.py

```diff
@@ -0,0 +1,45 @@
+from flojoy import TextBlob, flojoy
+from typing import Literal
+
+
+@flojoy
+def TEXT_CONCAT(
+    a: TextBlob,
+    b: TextBlob,
+    delimiter: Literal["space", "comma", "semicolon", "colon", "new line"] = "space",
+) -> TextBlob:
+    """The TEXT_CONCAT node takes in two TextBlob DataContainer type and cancatenates the text string.
+
+    Inputs
+    ------
+    a: TextBlob
+        The input text to be concatenated to input b
+
+    b: TextBlob
+        The input text to be concatenated to input a
+
+    Parameters
+    ----------
+    delimiter: "space" | "comma" | "semicolon" | "colon" | "new line", default="space"
+        Select the delimiter to place between two text.
+
+    Returns
+    -------
+    TextBlob
+       The text result from concatenation.
+    """
+
+    delim: str = ""
+    match delimiter:
```

Comment:
I think "none" should be an option just in case.

Reply:
Good point! Will reflect the changes